config_vsc.py

```diff
@@ -1,29 +1,42 @@
 import grp
 import os
+import time
 
 from py import builtin
 
-# use 'info' to log to syslog
-syslog_level = 'warning'
 
-perf_logging_format = 'reframe: ' + '|'.join(
-    [
-        'username=%(osuser)s',
-        'version=%(version)s',
-        'name=%(check_name)s',
-        'system=%(check_system)s',
-        'partition=%(check_partition)s',
-        'environ=%(check_environ)s',
-        'num_tasks=%(check_num_tasks)s',
-        'num_cpus_per_task=%(check_num_cpus_per_task)s',
-        'num_tasks_per_node=%(check_num_tasks_per_node)s',
-        'modules=%(check_modules)s',
-        'jobid=%(check_jobid)s',
-        'perf_var=%(check_perf_var)s',
-        'perf_value=%(check_perf_value)s',
-        'unit=%(check_perf_unit)s',
+standard_mode_options = [       
```

Comment:
this is specific for antwerpen, so can you rename it to e.g. `antwerpen_mode_options`?

Reply:
ok